server/src/models/Site.js

```diff
@@ -0,0 +1,9 @@
+const Model = require("./Model.js")
+
+class Site extends Model {
+    static get tableName() {
+        return "sites"
+    } 
```

Comment:
need to have matching model level jsonSchema validations for the constraints added to the migration! such as name, address, description, etc being required